src/main/webapp/app/course/competencies/taxonomy-select/taxonomy-select.component.html

```diff
@@ -3,8 +3,8 @@
     @for (taxonomy of competencyTaxonomy | keyvalue: keepOrder; track taxonomy; let i = $index) {
         <option [ngValue]="taxonomy.key">
             {{ indent[i] }}&boxur;
-            {{ 'artemisApp.competency.taxonomies.' + taxonomy.value | artemisTranslate }}
-            ({{ 'artemisApp.competency.keywords.' + taxonomy.value | artemisTranslate }})
+            {{ 'artemisApp.courseCompetency.taxonomies.' + taxonomy.value | artemisTranslate }}
+            ({{ 'artemisApp.courseCompetency.keywords.' + taxonomy.value | artemisTranslate }})
```

Comment:
not sure if we can get rid of the translate pipe here

Reply:
I'm open for suggestions

Reply:
maybe use 2 times <span>